### PR TITLE
Script to resize the images

### PR DESCRIPTION
**Pull request by a1exymoroz**
### What is this PR about?
- script to resize post images

### CODE REVIEW

File: tools/process-post-images/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            .catch(err => {
+              console.error(err);
+            });
+        });
+      }
+    });
+  });
+};
+
+updateSizeImage(sourceDir);
```
Comment (robinsonzimmermann) [inline tools/process-post-images/index.js]: Can't we reuse the existing script passing some parameters?

**Review by robinsonzimmermann — CHANGES_REQUESTED** [review]
Please add the directory with the processed images to .gitignore

File: tools/process-post-images/index.js (hunk 1 above)
Comment (a1exymoroz) [reply tools/process-post-images/index.js]: No. Because we need to recursively check all folders.

Comment (a1exymoroz) [reply tools/process-post-images/index.js]: Done

**Comment by robinsonzimmermann** [comment]
The images of related posts get broken when you navigate from one post to another. Can you take a look at it, please?

Other than that, looks good!

image

**Review by robinsonzimmermann — APPROVED** [review]

**Pull request merged**